app/Http/Livewire/Album.php

```diff
@@ -42,7 +42,7 @@ public function mount(AlbumModel $album, AlbumFactory $albumFactory)
 	 */
 	public function render()
 	{
-		switch (Configs::get_value('layout')) {
+		switch (Configs::getValueAsString('layout')) {
```

Comment:
```suggestion
		switch (Configs::getValueAsInt('layout')) {
```

Ans then the cases below should be integers as well?